data/windows/win_susp_add_sid_history.yml

```diff
@@ -0,0 +1,30 @@
+title: Addition of SID History to Active Directory Object
```

Comment:
Can we add a reference to where this rule was copied from?

Also, what license to these rules have?

Reply:
Clarified with the Sigma people. A link to the Sigma repo is good. The rules are all under: https://github.com/Neo23x0/sigma/blob/master/LICENSE.LGPL.txt

Reply:
addressed